_data-prepper/index.md

```diff
@@ -14,9 +14,9 @@ redirect_from:
 
 # Data Prepper
 
-Data Prepper is a server-side data collector capable of filtering, enriching, transforming, normalizing, and aggregating data for downstream analytics and visualization.
+Data Prepper is a server-side data collector capable of filtering, enriching, transforming, normalizing, and aggregating data for downstream analytics and visualization. Data Prepper is the preferred data ingestion tool for OpenSearch. It is recommended for most data ingestion use cases in OpenSearch and for processing large, complex datasets.
 
-Data Prepper lets users build custom pipelines to improve the operational view of applications. Two common uses for Data Prepper are trace and log analytics. [Trace analytics]({{site.url}}{{site.baseurl}}/observability-plugin/trace/index/) can help you visualize the flow of events and identify performance problems, and [log analytics]({{site.url}}{{site.baseurl}}/observability-plugin/log-analytics/) can improve searching, analyzing and provide insights into your application.
+With Data Prepper you can build custom pipelines to improve the operational view of applications. Two common uses for Data Prepper are trace analytics and log analytics. [Trace analytics]({{site.url}}{{site.baseurl}}/observability-plugin/trace/index/) can help you visualize events flow and identify performance problems. [Log analytics]({{site.url}}{{site.baseurl}}/observability-plugin/log-analytics/) can help improve searching, analyzing and give you deeper insights into your application.
```

Comment:
The last sentence needs some clarification. Improve search and analytics what or how?

Reply:
Does this read better? `Log analytics equips you with tools to enhance your search capabilities, conduct comprehensive analysis, and gain profound insights into your applications' performance and behavior.`

Reply:
Perfect!